src/aihwkit/nn/conversion.py

```diff
@@ -113,3 +118,46 @@ def convert_to_analog(
         module._modules[name] = new_mod  # pylint: disable=protected-access
 
     return module
+
+
+def convert_to_analog_mapped(
+        module: Module,
+        rpu_config: RPUConfigGeneric,
+        realistic_read_write: bool = False,
+        weight_scaling_omega: float = 0.0,
+) -> Module:
+    """Convert a given digital model to analog counter parts with tile
+    mapping.
+
+    Note:
+        The torch device (cuda/cpu) is inferred from the original
+        models parameters, however, if multiple torch
+        devices are used in a given module, the corresponding analog
+        module is not moved to any device.
+
+    Args:
+        module: The torch module to convert. All layers that are
+            defined in the ``conversion_map``.
+        rpu_config: RPU config to apply to all converted tiles.
+            Applied to all converted tiles.
+        realistic_read_write: Whether to use closed-loop programming
+            when setting the weights. Applied to all converted tiles.
+        weight_scaling_omega: If non-zero, applied weights of analog
+            layers will be scaled by ``weight_scaling_omega`` divided by
+            the absolute maximum value of the original weight matrix.
+
+            Note:
+                Make sure that the weight max and min setting of the
+                device support the desired analog weight range.
+
+    Returns:
```

Comment:
Change to:
Returns:
   Module where all the digital layers are replaced with analog mapped layers.